model/Security/Vocabularies/GenericVulnerabilitySeverityType.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# GenericVulnerabilitySeverityType
+
+## Summary
+
+TODO
+
+## Description
+
+A GenericVulnerabilitySeverityType is TODO
+
+## Metadata
+
+- name: GenericVulnerabilitySeverityType
+
+## Entries
+
+- low: TODOdescription
+- medium: TODOdescription
+- high: TODOdescription
+- critical: TODOdescription
+- noAssertion: TODOdescription
+
```

Comment:
Should none be added here?

Reply:
Removed this file, Defect team decided yesterday to move this assessment option to GA or 3.0.1.

Reply:
I'll remove the file and we can revert the commit for 3.1 or GA